docs/en/latest/api/api.md

```diff
@@ -21,351 +21,881 @@ title: API doc of Manager API.
 #
 -->
 
+## Description
```

Comment:
This document is too long. 
If there is an increase in API in the future, this article will become longer. 

I hope that the article will be divided into different articles according to the management object.